COLLABORATOR_GUIDE.md

```diff
@@ -122,12 +122,14 @@ Before landing pull requests, sufficient time should be left for input
 from other Collaborators. In general, leave at least 48 hours during the
 week and 72 hours over weekends to account for international time
 differences and work schedules. However, certain types of pull requests
-can be fast-tracked and may be landed after a shorter delay:
-
-* Focused changes that affect only documentation and/or the test suite.
-  `code-and-learn` and `good-first-issue` pull requests typically fall
-  into this category.
-* Changes that fix regressions.
+can be fast-tracked and may be landed after a shorter delay. For example:
+
+* Focused changes that affect only documentation and/or the test suite:
+  * `code-and-learn` tasks typically fall into this category.
+  * `good-first-issue` pull requests may also be suitable.
+* Changes that fix regressions:
+  * Regressions that break the workflow (red CI or broken compilation).
+  * Regressions that happen right before a release, or reported soon after.
```

Comment:
Maybe: "Fixes that need to be backported to a release line urgently."

If it's reported soon after a release and we decided to leave it till the next one, then it doesn't need to be fast-tracked IMO.

Reply:
I think this is just listing changes that "can be" fast-tracked, not necessarily "needs to be"?